pkg/controller/installation/core_controller.go

```diff
@@ -329,7 +340,7 @@ func mergeProvider(cr *operator.Installation, provider operator.Provider) error
 	if cr.Spec.KubernetesProvider == operator.ProviderNone {
 		cr.Spec.KubernetesProvider = provider
 	}
-	log.WithValues("provider", cr.Spec.KubernetesProvider).Info("Determined provider")
+	log.WithValues("provider", cr.Spec.KubernetesProvider).V(1).Info("Determined provider")
```

Comment:
I would like to use constants in this situation because I'm not sure/confident on what 'level' a 1 is and I think constants would help us have a better shared understanding when deciding what level a message should be at. (I'm not suggesting you change in this PR but think we should perhaps go through and make that change separately.)

Reply:
After some reading I think this should be 0 level. I think that is something we would always want to see in the logs.
https://github.com/go-logr/logr#how-do-i-choose-my-v-levels

Reply:
The problem with this one is that it's logged every loop of the controller, which actually happens a lot, even if nothing is happening. It adds a lot of noise and log spam. 

Wonder if we can ratelimit, or log it somewhere else that's not in the main loop.